build/merge-coverage.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-env node */
+const fs = require('fs');
+
+// Jest writes coverage after the process exits
+// so we need to poll and wait for all coverage files.
```

Comment:
What if there is an error and coverage is never written?

Reply:
The node process should just exit in this case, killing the CLI as well. I'll go ahead and include a timeout here though in case coverage never gets written for some reason.